docs/src/notation.md

```diff
@@ -0,0 +1,22 @@
+# Notation overview
+
+Since manifolds include a reasonable amount of elements and functions, the following list tries to keep an overview of used notation throughout `Manifolds.jl`.
+The order is alphabetically by name.
+They might be used in a plain form within the code or when referring to that code.
+This is for example the case the calligraphic symbols.
+
+Within the documented functions the utf8 symbols are used whenever possible,
+as long as that renders still in $\TeX$ within this documentation.
+
+| Symbol | Description | Also used | Comment |
+|:--:|:-------------- |:--:|:--- |
```

Comment:
I think it could be useful if this table contained Unicode symbols where they are supposed to be used now (like `\mathcal M`) so they can be copied and pasted when needed.

Reply:
Yes, I tried that, in Markdown that changed quite a few math formulas not to render (with ` `` ` delimiter they were transformed to `$` delimiters, with `$` delimiters they stayed unchanged. I think some UTF8 characters break MathJax or something. So to keep this table unified, I transformed all to just math until I see what the reason is. Also $\langle$ was rendered here rather strangely when using UTF8 (while in docstrings it seems to be nice and right).